### PR TITLE
Replace occurences of "modern dark" with spectrum

### PR DESCRIPTION
**Pull request by AObuchow**

### CODE REVIEW

**Comment by AObuchow** [comment]
Merging as this seems okay to me (briefly tested... kind of risky, but that's what user feedback is for)

File: com.aobuchow.themes.spectrum/plugin.xml [hunk 1]
```diff
@@ -5,38 +5,24 @@
          point="org.eclipse.e4.ui.css.swt.theme">
       <theme
             basestylesheeturi="css/Spectrum.css"
-            id="modern.dark.theme.id"
```
Comment (PyvesB) [inline com.aobuchow.themes.spectrum/plugin.xml:8]: @AObuchow this change switched the theme to a light one on Windows. Remember https://github.com/AObuchow/Eclipse-Spectrum-Theme/issues/1#issuecomment-640028240 😉

Comment (AObuchow) [reply com.aobuchow.themes.spectrum/plugin.xml:8]: OOPS I totally forgot about that!!! Thank you for letting me know, I’ll fix that right away :)

Comment (AObuchow) [reply com.aobuchow.themes.spectrum/plugin.xml:8]: Fixed in https://github.com/AObuchow/Eclipse-Spectrum-Theme/commit/ee2cf5fd297f96e4d7f32c0fb41779fe781e20ce

Also: 100th commits wooo

**Pull request merged**